composer.json

```diff
@@ -15,11 +15,11 @@
     "require": {
         "php": ">=5.5",
         "zendframework/zend-escaper": "~2.5",
-        "zendframework/zend-stdlib": "~2.5"
+        "zendframework/zend-stdlib": "dev-develop as 2.8.0"
```

Comment:
@weierophinney I think we can tag 2.8 of zend-stdlib and 2.6 of zend-config, we have a ton of usage around with "dev-develop as".